history-of-quakers/modernized/14-chapter-12.adoc

```diff
@@ -849,6 +880,9 @@ At Dover, in New England, Anne Coleman, Mary Tomkins, and Alice Ambrose,
 were sentenced to very cruel whipping, only for being come there:
 the warrant was as follows:
 
+[.embedded-content-document.legal]
+--
+
 To the constables of Dover, Hampton, Salisbury, Newbury, Rowley, Ipswich, Wenham, Linn,
```

Comment:
Should this have a letter head or a [salutation]? I had a hard time with these today, I can't remember how I've done them. :)

Reply:
This is a weird one.  It's very long and strange.  Could be a salutation, or a letter-heading, or just a regular paragraph.  I don't blame you for being confused on this one.  I guess I think maybe let's do a [.letter-heading] is my best thought.